packages/jupyter-chat/src/widgets/chat-widget.tsx

```diff
@@ -19,33 +18,21 @@ export class ChatWidget extends ReactWidget {
     this.title.icon = chatIcon;
     this.title.caption = 'Jupyter Chat'; // TODO: i18n
 
-    this._model = options.model;
-    this._themeManager = options?.themeManager || null;
-    this._rmRegistry = options.rmRegistry;
+    this._chatOptions = options;
   }
 
   /**
    * Get the model of the widget.
    */
   get model(): IChatModel {
-    return this._model;
+    return this._chatOptions.model;
   }
 
   render() {
-    return (
-      <Chat
-        model={this._model}
-        themeManager={this._themeManager}
-        rmRegistry={this._rmRegistry}
-      />
-    );
+    // The model need to be passed, otherwise it is undefined in the widget in
+    // the case of collaborative document.
+    return <Chat {...this._chatOptions} model={this._chatOptions.model} />;
```

Comment:
The `model` need to be passed to the widget, the `chatOptions` is not enough in the case of collaborative document.
Probably because the model is a getter with collaborative document:

https://github.com/jupyterlab/jupyterlab/blob/130089eddb330a44f7f9d496b1e64eddcb497d6b/packages/docregistry/src/context.ts#L140-L142